shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -78,7 +78,14 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
 
   def hasValidApi: Boolean = {
     val maybeApiSplits = splits.find(_.source == SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages)
-    val hasLiveSource = apiFlight.FeedSources.contains(LiveFeedSource)
+    val totalPaxSourceIntroductionMillis = 1655247600000L // 2022-06-15 midnight BST
```

Comment:
why are considering this date for paxSourceAvailable ?

Reply:
It's the date we released the TotalPaxSource work. Prior to this date we didn't have this data so we can't check it